### DISCUSSION TITLE
Dynamics of Dications

### DISCUSSION THREAD

**Discussion by PARobertson** [qcxms/QCxMS#67] [General]
Hi,

We work on reaction dynamics of multiply charged ions formed from EI. We have experimental methods for measuring product scattering distributions for all ions per cycle and way to find correlated ions and their relative velocity vectors. As you can imagine, there are all sorts of weird rearrangement reactions that happen to highly energetic ions, and we have developed a pretty good empirical understanding what sort of mechanisms are happening from our experiments. However, it's very hard to convince a reviewer without some theory to back it up. 

For example, we have pretty clear experimental evidence that the furan dication will ejected [C2H2+] and if the [C2H2O+] co-fragment has enough internal energy, it will rearrange to form [CH2=C=O+], which then dissociates to form [CH2+], but no way to show what that sort of rearrangement might look like, or what timescales it might occur on, which hinders our ability to publish.

The usual approach in reaction dynamics theory is to do full ab initio non-adiabatic dynamics, but the sorts of molecules we study are too large to perform these on, and since we do EI (rather than photoionization), the states involved are ambiguous. We've recently explored using BOMD to search for trajectories that lead to what products we observe, and in doing so I have found QCxMS. The IEE method is a perfect analogy for how we understand the initial post-ionization dynamics to occur, leaving us with a highly internally excited radical dication. 

On face value, QCxMS could be perfectly suited to our problem, but it would require some tweaks to allow multply charged radical cations, multiplicity selection and a way to save trajectories that lead to products of interest. It would also be nice to be able to know the IEE assigned to the trajectory, as that could guide our ability to assign which state(s) might be involved.

Anyway, this is clearly a non-trivial exercise, but I'd be interested to hear if anyone has any thoughts on it's viability or how to proceed? 

Cheers,
Patrick

**Comment by gorges97** [comment]
It would be an interesting feature to allow multiply charged ions in the EI mode to QCxMS  but there are currently no plans to develop this. 

However, it is possible to calculate multiply charged ions in the CID mode. The "temprun" mode with appropriate energy settings (controllable via keyword esi) could already be sufficient to generate the fragments observed in your experiment. Since the trajectories are printed by QCxMS you can look at the corresponding reaction mechanism. The IEE of the respective MD simulation can also be found in the output file ("final inner E " given in eV).
